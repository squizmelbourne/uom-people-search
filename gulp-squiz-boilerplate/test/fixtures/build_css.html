<!-- build:css test/.tmp/styles/main.scss { "media": "(min-width: 60em)" } -->
    <!-- import:css test/fixtures/styles/main.scss -->
    <!-- import:css test/fixtures/styles/concat.css -->
<!-- endbuild -->