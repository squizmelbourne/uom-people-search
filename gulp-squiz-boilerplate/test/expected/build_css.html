<link href="test/.tmp/styles/main.scss" rel="stylesheet" media="(min-width: 60em)" />